ironfish-cli/src/commands/peers/add.ts

```diff
@@ -0,0 +1,61 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { DEFAULT_WEBSOCKET_PORT, parseUrl } from '@ironfish/sdk'
+import { IronfishCommand } from '../../command'
+import { RemoteFlags } from '../../flags'
+
+export class AddCommand extends IronfishCommand {
+  static description = `Add a peer to the list of candidates`
+
+  static args = [
+    {
+      name: 'url',
```

Comment:
It's not too important, but I'd lean toward calling this a `socket`, `address`, or `endpoint` instead, since a URL includes a path + protocol

Reply:
alright yea I agree